pkg/node/local.go

```diff
@@ -180,6 +182,27 @@ func StartLocalNode(
 		localDataExists = false
 	}
 
+	// set node configs
```

Comment:
I think it would be cleaner for the caller of `StartLocalNode` to set the `nodeConfig` parameter properly for their use case rather than always overriding it with specific settings here. This allows the function to remain more general use for the future, for instance if we want to a start a local node that is going to become a primary network validator.

Reply:
I think we should also only ever serialize the `config` struct to start a node, rather than creating a `Config` instance, serializing it as JSON, then deserializing it to override certain keys, etc. Using the AvalancheGo defined type will be less error prone in the event of any future changes to the config structure.

Reply:
changed the Start() param to be a map[string]interface{} , and serialize it inside
I believe the avalanchego defined type (if it is the one define under avalanchego/node/config.go)
is not a good fit as we are just trying to set flags for the avalanchego command line, and it seems 
not direct to map from the internal Config type to the desired flags. Unless I am missing something.